src/vulkan-renderer/exceptions/vk_exception.cpp

```diff
@@ -75,7 +73,7 @@ namespace inexor::vulkan_renderer {
     return "Unknown";
```

Comment:
Could be in the default case of the switch case. No idea what we use.

Reply:
Clang-tidy will complain if we don't offer this explicitely.